luaui/Widgets/gui_prospector.lua

```diff
@@ -1,7 +1,7 @@
 function widget:GetInfo()
 	return {
 		name = "Prospector",
-		desc = "Tooltip for amount of metal extracted when placing metal extractors.",
+		desc = "Tooltip for amount of metal extracted when placing mexes.",
```

Comment:
perhaps it would be good to mention it's for metalmaps where we can't just use spots for that